recipes/dcmtk/all/conandata.yml

```diff
@@ -6,7 +6,10 @@ patches:
   "3.6.7":
     - patch_file: "patches/3.6.7-0001-cmake-robust-deps-handling.patch"
       patch_description: "CMake: robust discovery with find_package() and use imported targets"
-      patch_type: conan
+      patch_type: "conan"
     - patch_file: "patches/3.6.7-0002-cmake-check-openssl-symbol.patch"
       patch_description: "CMake: fix OpenSSL compatibility checks"
-      patch_type: conan
+      patch_type: "conan"
+    - patch_file: "patches/3.6.7-0003-ambiguous-overload-operator-equal.patch"
```

Comment:
Hi! Thanks a lot for your contribution - As this is a C++ change, it needs a bit more consideration. Has this been reported upstream? I'm guessing no as theres no patch_source field.

After checking the code changes they look good to me, but would still appreciate an lib author confirmation on their side if you could get one :)

Reply:
I'll do when I have time. There is no way to report issues in dcmtk repo, only PR.

Reply:
Thanks a lot :) Note that I've checked and the changes look valid to me, but I'd like to still have a second opinion, in unlikely case that it somehow is not correct :)

Thanks a lot for your work

Reply:
yep, I'll submit a PR upstream today or tomorrow.

Reply:
I've submitted https://github.com/DCMTK/dcmtk/pull/88

Reply:
Turn to draft, it might be fixed by defining other operator==

Reply:
Upstream seems to agree with the new patch I've submitted (define additional `operator==`): https://github.com/DCMTK/dcmtk/pull/88#issuecomment-1760322478